### PR TITLE
Removed boxed codependence. Added sample id for each box.

### LINKED ISSUE #66: BUG: Card without label should not be collapsible.

### PR DESCRIPTION
**Pull request by maju116**
Closes #66

The problem was that, if at least one box had a collapse button then javascript would work on every box in app. Fixed by adding unique id for each `box`.

### CODE REVIEW

File: R/box.R [hunk 1]
```diff
@@ -16,6 +16,7 @@
 box <- function(..., title = NULL, color = "", ribbon = TRUE, title_side = "top right", collapsible = TRUE, width = 8) {
   verify_value_allowed("color", c("", ALLOWED_COLORS))
   verify_value_allowed("title_side", if (ribbon) ALLOWED_BOX_SIDES_RIBBON else ALLOWED_BOX_SIDES_NONRIBBON)
+  box_id <- paste0("box_", sample(1:10000000, 1))
```
Comment (filipstachura) [inline R/box.R]: This is going to fail once in 10000000 by assigning the same id to 2 boxes. Can we use a counter instead?

Comment (maju116) [reply R/box.R]: Added `id` argument, if `NULL` then random.

File: R/tab_box.R [hunk 2]
```diff
@@ -16,7 +17,7 @@
 #'          list(menu = "Second Tab", content = "This is second tab")
 #'        ))
 tab_box <- function(tabs, title = NULL, color = "", ribbon = TRUE,
-                    title_side = "top right", collapsible = TRUE, width = 8) {
+                    title_side = "top right", collapsible = TRUE, width = 8, id = NULL) {
   box(shiny.semantic::tabset(tabs), title = title, color = color, ribbon = ribbon, title_side = title_side,
```
Comment (filipstachura) [inline R/tab_box.R]: I think `id` should be passed along to `box`.

Comment (maju116) [reply R/tab_box.R]: You're right, sorry

Comment (maju116) [reply R/tab_box.R]: done

**Pull request merged**